readme.md

```diff
@@ -1,12 +1,12 @@
-# Bitbucket Server integration
+# Bitbucket integration
```

Comment:
How can I as a user know that this is **not** integrating with Bitbucket cloud?

Reply:
I did explicitly change to Bitbucket Data Center in some places (e.g. line 9 below) for this reason. I think I'll do the same for the title here as you've called out.